src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +74,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
```

Comment:
Please do not change the indentation level of switch. The original style is correct. See our [coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html).

<img width="663" alt="image" src="https://user-images.githubusercontent.com/18503402/45559106-2a49d900-b874-11e8-99de-24780c40f18d.png">

Reply:
Alright noted!